components/tabs/src/tabs.udl

```diff
@@ -1,6 +1,10 @@
 [Custom]
 typedef string TabsGuid;
 
+// a local timestamp, a custom type to avoid using the int directly.
+[Custom]
+typedef i64 Timestamp;
```

Comment:
I thought we had [a built-in `Timestamp` type](https://github.com/mozilla/uniffi-rs/tree/f15901cf54b6d68be8ee643cf52b3296d16c11b2/fixtures/uniffi-fixture-time) in UniFFI—would that work here?

Reply:
Not really - we don't use the underlying types backing that. TBH, I think UniFFI should deprecate those types now that we have custom types.